Sources/Clappr/Classes/Plugin/Core/UICorePlugin.swift

```diff
@@ -9,6 +9,16 @@ open class UICorePlugin: SimpleCorePlugin, UIPlugin {
         }
     }
     
+    #if os(tvOS)
+    func updateFocus(userInfo: EventUserInfo) {
+        core?.trigger(.requestFocusUpdate, userInfo: userInfo)
+    }
+    
+    func updateFocus() {
```

Comment:
I consider that this method should not be provided: it only works if `func updateFocus(userInfo: EventUserInfo)` has already been called and requires that a strong reference to an existing `Plugin` be kept by the event receiver.

Reply:
The behavior of update focus is to trigger the update focus of the Focus Engine from tvOS and it will change the focus depending of the state of the view, we can move this for other layer if it makes more sense.

Reply:
Done